azurerm/resource_arm_key_vault_test.go

```diff
@@ -648,3 +670,78 @@ resource "azurerm_key_vault" "test" {
 }
 `, rInt, location, rInt)
 }
+
+func testAccAzureRMKeyVault_accessPolicyUpperLimit(rInt int, location string) string {
+	
+	var storageAccountConfigs string
+	var accessPoliciesConfigs string
+	var dependsString string
+        var maxPolicies = 68
+
+	for i := 50; i <= maxPolicies; i++ {
+		storageAccountConfigs += testAccAzureRMKeyVault_generateStorageAccountConfigs(i)
+		accessPoliciesConfigs += testAccAzureRMKeyVault_generateAccessPolicyConfigs(i)
+		dependsString += fmt.Sprintf(`"azurerm_storage_account.testsa%d"`, i)
+		if i < maxPolicies {
+		dependsString += ", "
+		}
+		
+	}
+
+	return fmt.Sprintf(`
+data "azurerm_client_config" "current" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_key_vault" "test" {
+  name                = "vault%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  tenant_id           = "${data.azurerm_client_config.current.tenant_id}"
+
+  sku {
+    name = "premium"
+  }
+%s
+  depends_on = [%s]
+}
+
+%s
+`, rInt, location, rInt, accessPoliciesConfigs, dependsString, storageAccountConfigs)
+}
+
+func testAccAzureRMKeyVault_generateStorageAccountConfigs(accountNum int) string {
+	return fmt.Sprintf(`
+resource "azurerm_storage_account" "testsa%d" {
+    name                      = "testsa%d"
```

Comment:
could we use a random short string here instead of a number? this will not be unique enough

Reply:
Yep... I did both, for good measure.